storeshare_api/config/database.yml

```diff
@@ -19,8 +19,8 @@ development:
   encoding: unicode
   database: storeshare_development
   pool: 5
-  username: yigit
-  password:
+  username: kutaykarakas
```

Comment:
should this credential change be committed to main? (or is it a local change only)